docs/integrations/sources/square.md

```diff
@@ -79,4 +79,5 @@ Some Square API endpoints has different page size limitation
 
 | Version | Date       | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
-| 0.1.0   | 2021-06-30 | [4439](https://github.com/airbytehq/airbyte/pull/4439) | Initial release supporting the Square API |
\ No newline at end of file
+| 0.1.0   | 2021-06-30 | [4439](https://github.com/airbytehq/airbyte/pull/4439) | Initial release supporting the Square API |
```

Comment:
I think latest version should be the first in table

Reply:
Good point. Thanks